scipy/cluster/tests/test_hierarchy.py

```diff
@@ -71,40 +71,34 @@
 
 skip_xp_backends = pytest.mark.skip_xp_backends
 xfail_xp_backends = pytest.mark.xfail_xp_backends
-use_linkage = skip_xp_backends(cpu_only=True, exceptions=["jax.numpy"],
-                               reason="linkage() invokes Cython code")
+use_linkage = skip_xp_backends(cpu_only=True, reason="linkage() invokes Cython code")
```

Comment:
All this needs to be migrated to `xp_capabilities`. I'd rather do it downstream of this in order to minimize rework.